bitnami/memcached/values.yaml

```diff
@@ -26,17 +26,28 @@ image:
   # pullSecrets:
   #   - myRegistryKeySecretName
 
+## Extra environment vars to pass.
+## ref: https://github.com/bitnami/bitnami-docker-memcached#configuration
+extraEnv: []
+
 ## String to partially override memcached.fullname template (will maintain the release name)
 ##
 # nameOverride:
 
+## Number of containers to run
+replicaCount: 1
+
 ## String to fully override memcached.fullname template
 ##
 # fullnameOverride:
 
 # Cluster domain
 clusterDomain: cluster.local
 
+## Memcahed architecture. Allowed values: standalone or high-availability
+##
+architecture: standalone
```

Comment:
If you check the helpers of InfluxDB (lines 280 to 302): https://github.com/bitnami/charts/blob/master/bitnami/influxdb/templates/_helpers.tpl
We created a "validateValues" macro that can include some validations. For instance, we create a validation (`validateValues.architecture`) to ensure that only valid values are set on that parameter.

Could we do sth similar here?  

This macro is used in the NOTES.txt: https://github.com/bitnami/charts/blob/master/bitnami/influxdb/templates/NOTES.txt#L140